ms2deepscore/MS2DeepScoreData.py

```diff
@@ -0,0 +1,166 @@
+import numpy as np
+from tqdm import tqdm
+from ms2deepscore.spectrum_binning_fixed import create_peak_list_fixed
+from ms2deepscore.spectrum_binning_fixed import set_d_bins_fixed
+from ms2deepscore.spectrum_binning_fixed import unique_peaks_fixed
+from ms2deepscore.utils import create_peak_dict
+
+
+class MS2DeepScoreData:
```

Comment:
Good ideas, thanks a lot Sven. I renamed it.

Reply:
I also agree that extracting inchikeys in maybe not what one would expect here, but I wanted to have all necessary information about the data in one place (also to better ensure that the order is right etc.). It could maybe be cleaner to just keep the inchikey as part of each binned spectrums (which currently is still just a `dict`) ?

Reply:
for the API changes (renaming and making methods private): I would skip this for now. I think it is a very good idea actually, but I realize had a slightly different API in mind and hence would rather work on it after first merging this.

Reply:
Cool! Regarding making the `inchikey` a part of the binned spectrum, that might be a good idea. In that case I suggest outputting a list of `BinnedSpectrum` objects that have `spectrum` and `inchi_key` properties. This clashes with the `n_spectrums x n_bins` idea though.